scene/3d/skeleton_3d.cpp

```diff
@@ -1097,6 +1101,14 @@ void Skeleton3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("physical_bones_start_simulation", "bones"), &Skeleton3D::physical_bones_start_simulation_on, DEFVAL(Array()));
 	ClassDB::bind_method(D_METHOD("physical_bones_add_collision_exception", "exception"), &Skeleton3D::physical_bones_add_collision_exception);
 	ClassDB::bind_method(D_METHOD("physical_bones_remove_collision_exception", "exception"), &Skeleton3D::physical_bones_remove_collision_exception);
+
+	ADD_GROUP("Deprecated", "");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "animate_physical_bones"), "set_animate_physical_bones", "get_animate_physical_bones");
```

Comment:
Would it be better to hide this property from the inspector? I only need it to be exposed to scripting to prevent breaking binary compatibility.

```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "animate_physical_bones", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NO_EDITOR), "set_animate_physical_bones", "get_animate_physical_bones");
```

Reply:
For compatibility with past projects, I think it is fine to have this show in the editor. Since some initialization process changes were included in this PR to make this public, so I believe it works without any problem.